### PR TITLE
Random-Move-Implementation

### PR DESCRIPTION
**Pull request by datGUUYY**

### CODE REVIEW

File: idk/src/main/java/com/treydev/idk/attack/Move.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 import com.treydev.idk.critter.Critter;
 import com.treydev.idk.support.Element;
+import com.treydev.idk.support.NameGenerator;
```
Comment (dmckinstry) [inline idk/src/main/java/com/treydev/idk/attack/Move.java:9]: Delete unused import

File: idk/src/main/java/com/treydev/idk/critter/Species.java [hunk 2]
```diff
@@ -75,12 +75,12 @@ public static Species GenRandomSpecies(int individualStatSize, Element primaryEl
         int SDPercent = random.nextInt(100);
         int HPPercent = random.nextInt(100);
         int total = AtkPercent + DefPercent + SAPercent + SpdPercent + SDPercent + HPPercent;
-        int baseAttack = statpoolSize * AtkPercent / total;
-        int baseDefense = statpoolSize * DefPercent / total;
-        int baseSpecial = statpoolSize * SAPercent / total;
-        int baseSpeed = statpoolSize * SpdPercent / total;
-        int baseSDefence = statpoolSize * SDPercent / total;
-        int baseHPStat = statpoolSize * HPPercent / total;
+        int baseAttack = statpoolSize * AtkPercent / total + 1;
```
Comment (dmckinstry) [inline idk/src/main/java/com/treydev/idk/critter/Species.java:78]: Inconsistent spaces in code.  Consider running the format code in VS Code.

**Review by dmckinstry — APPROVED** [review]
A couple of minor things to consider.

File: idk/src/main/java/com/treydev/idk/attack/Move.java (hunk 1 above)
Comment (datGUUYY) [reply idk/src/main/java/com/treydev/idk/attack/Move.java:9]: All of these seem to be used.

**Pull request merged**